counterpoint/lib/attribute.ts

```diff
@@ -684,7 +684,26 @@ export class Attribute<
     this.valueFn = undefined;
     return this;
   }
-
+  /**
+   * Sets transform of an attribute to a new function, can be applied to an Attribute,
+   * a Mark, or a MarkRenderGroup
+   * @param attrToModify attribute to modify
+   * @param newFunc new function to set transform to
+   */
+  setTransform(
+    attrToModify: Attribute<
```

Comment:
The `attrToModify` parameter doesn't make sense here because the `Attribute` can only modify itself. I like the consistency across the three classes but I don't think it's necessary in this case, so we can remove `attrToModify` for the Attribute only.